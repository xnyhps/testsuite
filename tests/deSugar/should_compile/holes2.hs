{-# LANGUAGE TypeHoles #-}

module Main where

main = return ()

f = show _
