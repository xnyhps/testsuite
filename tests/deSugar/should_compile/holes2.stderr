
holes2.hs:7:5: Warning:
    No instance for (Show a0) arising from a use of `show'
    The type variable `a0' is ambiguous
    Possible fix: add a type signature that fixes these type variable(s)
    Note: there are several potential instances:
      instance Show Double -- Defined in `GHC.Float'
      instance Show Float -- Defined in `GHC.Float'
      instance (Integral a, Show a) => Show (GHC.Real.Ratio a)
        -- Defined in `GHC.Real'
      ...plus 23 others
    In the expression: show _
    In an equation for `f': f = show _

holes2.hs:7:10: Warning:
    Found hole `_' with type a0
    Where: `a0' is an ambiguous type variable
    In the first argument of `show', namely `_'
    In the expression: show _
    In an equation for `f': f = show _
